CMakeLists.txt

```diff
@@ -108,6 +108,7 @@ set(CPACK_PACKAGING_INSTALL_PREFIX "/${CPACK_PACKAGE_NAME}")
 set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_CURRENT_SOURCE_DIR}/dist/LICENSE")
 set(CPACK_PACKAGE_INSTALL_DIRECTORY "${CPACK_PACKAGE_NAME}")
 set(CPACK_GENERATOR TGZ;ZIP)
+set(CPACK_PACKAGE_CHECKSUM "SHA512")
```

Comment:
Isn't CMake nice!

Reply:
The ticket only asked for a SHA-256 hash, is 512 consistent with Geode?  Let's get the ticket updated accordingly.

Reply:
512 is fine...and actually faster on most cpus :-)

Reply:
GPG signature gen shouldn't be automated, per @metatype.  Release will be signed by release manager personally during release process. 

With respect to hash, I went with 512 because that's what the Geode source release uses.